bower.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This is odd that this is being submitted, this looks like the same version as I submitted.